### PR TITLE
Fix #373 config.json cacheControl

### LINKED ISSUE #373: config.json cacheControl
remove 
etagCacheControlHeader

rename
immutableCacheControlHeader to cacheControl

### PR DESCRIPTION
**Pull request by ComLock**

### CODE REVIEW

File: src/main/java/lib/enonic/libStatic/IoService.java [hunk 1]
```diff
@@ -94,6 +79,7 @@ public void initialize( BeanContext context )
     {
         this.resourceServiceSupplier = context.getService( ResourceService.class );
         this.parentResourceKey = context.getResourceKey();
+        this.bundle = FrameworkUtil.getBundle( this.getClass() );
```
Comment (rymsha) [inline src/main/java/lib/enonic/libStatic/IoService.java:82]: this is not in use

File: src/main/resources/lib/enonic/static/constants.ts [hunk 2]
```diff
@@ -1,27 +1,98 @@
...
+  IF_NONE_MATCH = 'if-none-match'
+}
 
-export const GETTER_ROOT = 'static';
+export const enum HTTP2_RESPONSE_HEADER {
+  CACHE_CONTROL = 'cache-control',
+  ETAG = 'etag',
+}
+
+export const enum RESPONSE_CACHE_CONTROL {
```
Comment (rymsha) [inline src/main/resources/lib/enonic/static/constants.ts:69]: I changed values in specs (again 😔 )

File: src/main/resources/lib/enonic/static/service/serviceUrlRootViaAssetUrl.ts [hunk 3]
```diff
@@ -8,10 +8,12 @@ import { assetUrl as getAssetUrl } from '/lib/xp/portal';
 const SERVICE_NAME = 'static';
 
 export function serviceUrlRootViaAssetUrl({
```
Comment (rymsha) [inline src/main/resources/lib/enonic/static/service/serviceUrlRootViaAssetUrl.ts:10]: dead function?

File: src/main/java/lib/enonic/libStatic/IoService.java (hunk 1 above)
Comment (ComLock) [reply src/main/java/lib/enonic/libStatic/IoService.java]: Fixed

File: src/main/resources/lib/enonic/static/constants.ts (hunk 2 above)
Comment (ComLock) [reply src/main/resources/lib/enonic/static/constants.ts:69]: Fixed

File: src/main/resources/lib/enonic/static/service/serviceUrlRootViaAssetUrl.ts (hunk 3 above)
Comment (ComLock) [reply src/main/resources/lib/enonic/static/service/serviceUrlRootViaAssetUrl.ts]: Yes, removed

**Review by rymsha — APPROVED** [review]

**Review by alansemenov — APPROVED** [review]

**Pull request merged**